WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeOptionsViewModel.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import Yosemite
+
+/// Provides view data for Add Attributes, and handles init/UI/navigation actions needed.
+///
+final class AddAttributeOptionsViewModel {
+
+    typealias Section = AddAttributeOptionsViewController.Section
+    typealias Row = AddAttributeOptionsViewController.Row
+
+    private(set) var newAttributeName: String?
+    private(set) var attribute: ProductAttribute?
```

Comment:
nit: after looking at `title = viewModel.newAttributeName ?? viewModel.attribute?.name` in `AddAttributeOptionsViewController`, maybe we can just have a public property like `title` in the view model that is set by the two initializers? this way, we can encapsulate the title logic in the view model and make it testable